executor-core/src/main/java/net/spals/appbuilder/executor/core/DefaultExecutorServiceFactory.java

```diff
@@ -25,15 +29,16 @@
 class DefaultExecutorServiceFactory implements ExecutorServiceFactory {
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultExecutorServiceFactory.class);
 
+    private final Map<Key, ExecutorService> executorServices = new HashMap<>();
+
+    private final Tracer tracer;
+
     @Configuration("executorService.registry.shutdown")
     private volatile Long shutdown = 1000L;
 
     @Configuration("executorService.registry.shutdownUnit")
     private volatile TimeUnit shutdownUnit = TimeUnit.MILLISECONDS;
```

Comment:
How does a configuration work for an Enum?

Reply:
As I recall, the Typesafe Config library does a string-based lookup on name to resolve enum values. But can't remember 100%